rotkehlchen/exchanges/cryptocom.py

```diff
@@ -237,35 +268,22 @@ def _deserialize_trade(self, raw_result: dict[str, Any]) -> list[SwapEvent]:
          - UnknownAsset
          - UnsupportedAsset
         """
-        # Parse the trading pair (e.g., "BTC_USDT")
-        pair = raw_result['instrument_name']
-        base_asset_str, quote_asset_str = pair.split('_')
-
-        base_asset = asset_from_cryptocom(base_asset_str)
-        quote_asset = asset_from_cryptocom(quote_asset_str)
-
-        is_buy = raw_result['side'] == 'BUY'
-        amount = deserialize_fval(raw_result['traded_quantity'])
-        rate = deserialize_fval(raw_result['traded_price'])
-        fee = deserialize_fval_or_zero(raw_result.get('fee', '0'))
-        fee_currency = raw_result.get('fee_currency', '')
-
+        # Parse the instrument name (e.g., "BTC_USDT")
+        base_asset_str, quote_asset_str = raw_result['instrument_name'].split('_')
+        fee = deserialize_fval_force_positive(raw_result.get('fees', '0'))
```

Comment:
```suggestion
        fee = deserialize_fval_force_positive(fee_str) if fee_str := raw_result.get('fees') is not None else None
```

adjust the usage below